velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemTest.cpp

```diff
@@ -60,6 +60,40 @@ TEST_F(S3FileSystemTest, writeAndRead) {
   readData(readFile.get());
 }
 
+TEST_F(S3FileSystemTest, viaRegistry) {
```

Comment:
These tests are present in `S3FileSystemRegistrationTest.cpp`. Is there any reason to add them here?

Reply:
CC: @xiaoxmeng

Reply:
It is a result of reverting of an old change. I didn't notice they have been moved.